services/movie.py

```diff
@@ -28,13 +33,14 @@
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Code Style: It's a good practice to clearly define the type of elements in the list. Instead of `list` you can use `list[int]` to specify that the list contains integers.